packages/entities/entities-vaults/src/locales/en.json

```
@@ -343,27 +343,31 @@
         "description": "Access secrets stored in Conjur Key Vault for Kong Gateway configuration.",
         "fields": {
           "endpoint_url": {
-            "label": "Endpoint URL"
+            "label": "Conjur Server",
+            "tooltip": "The CyberArk Conjur backend url. Accepts http or https."
           },
           "login": {
-            "label": "Login"
+            "label": "Workload Identity",
+            "tooltip": "Identity of the workload or host authenticating to Conjur. In this case Kong is the workload/host."
           },
           "api_key": {
-            "label": "API Key"
+            "label": "API Key",
+            "tooltip": "The API key issued to the of the workload or host"
```

Seems we are missing something:
```suggestion
            "tooltip": "The API key issued to the {{what?}} of the workload or host"
```